scripts/freebsd-ci-test.sh

```diff
@@ -290,6 +292,7 @@ test_sccache_dist_02()
 
 cleanup()
 {
+	trap - EXIT INT HUP
```

Comment:
could you please add a comment here?

Reply:
@sylvestre I'll clean it up a bit so it's clear how it works (please ignore the intermediate commit)

Reply:
@sylvestre Code should be more obvious now.

Let's wait for CI builds to finish.